### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by ardiadrianadri**
Ok guys, at the end it will be 9 steps because I think we need a final section for the conclusions. But I promise you they will only be 9. This is the eighth. One more and the first version of the workshop will be finished and ready to show

### CODE REVIEW

File: src/app/common/marvel-elements.ts [hunk 1]
```diff
@@ -1,6 +1,25 @@
 import { BaseElement } from './base-element';
 
+/**
+ * All Marvel elements which does not fit in a hero definition
```
Comment (oleks-fedotov) [inline src/app/common/marvel-elements.ts]: which 'do' not fit

File: src/app/common/pag-event.ts [hunk 2]
```diff
@@ -1,4 +1,22 @@
+/**
+ * Definition of a page for the table
```
Comment (oleks-fedotov) [inline src/app/common/pag-event.ts]: Probably the name of the interface can be changed, to match the comment?

Comment (ardiadrianadri) [reply src/app/common/pag-event.ts]: Check if you like more the new comment

**Review by bytelovers — COMMENTED** [review]
In my opinion, it's ok

**Pull request merged**